doc/mgr/hello.rst

```diff
@@ -6,34 +6,47 @@ This is a simple module skeleton for documentation purposes.
 Enabling
 --------
 
-The *hello* module is enabled with::
+Run the following command to enable the ``hello`` module: 
```

Comment:
Hello, hello, hello!
What a wonderful word Hello

Reply:
I know this movie. That's David Paymer.